icons.go

```diff
@@ -101,15 +116,15 @@ func getUploadURL(query iconCandidateQuery, appIcons []appIconCandidateURL) ([]a
 
 		body, err := ioutil.ReadAll(resp.Body)
 		if err != nil {
-			return fmt.Errorf("failed to read respnse body, error: %s", err)
+			return fmt.Errorf("failed to read response body, error: %s", err)
 		}
 
 		if resp.StatusCode != http.StatusCreated {
 			return fmt.Errorf("invalid status code: %d, headers: %s, body: %s", resp.StatusCode, resp.Header, body)
 		}
 
 		decoded := map[string][]appIconCandidateURL{
```

Comment:
You still no need to initialize this var: https://www.socketloop.com/tutorials/golang-how-to-convert-json-string-to-map-and-slice